src/tests/data-test/ApodDatasource.test.ts

```diff
@@ -0,0 +1,112 @@
+import "mocha";
+import { getConnection } from "typeorm";
+import { getApod, setApod } from "../../data/datasource/Apod.datasource";
+import { expect } from "chai";
+import { Apod } from "../../data/entity/Apod.entity";
+
+function fakeApod() {
+  let apod = new Apod();
+  apod.url = "www.apod.com";
+  apod.title = "Death Star";
+  apod.explanation = "That's no moon.";
+  apod.media_type = "image";
+  return apod;
+}
+
+before(async () => {});
+
+beforeEach(async () => {});
+
+afterEach(async () => {
+  const connection = await getConnection();
+  await connection.dropDatabase();
+  await connection.synchronize();
+});
```

Comment:
Embora seja um pouco mais intuitivo limpar as tabelas (ou dropar o banco) depois do teste rodar eu sugiro fazer isso no `beforeEach` porque, caso você queira visualizar o banco de dados após um teste que falhou, terá como. Então ajuda a debugar. 

Ainda, não precisa do `before` e `beforeEach` se não for usá-los.